packages/block-editor/src/components/link-control/index.js

```diff
@@ -118,9 +118,10 @@ function LinkControl( {
 			'is-loading': isLoading,
 		} );
 
+		const searchText = inputValue.trim();
```

Comment:
Wondering if we should use [lodash trim function](https://lodash.com/docs/#trim) instead of the method.

Reply:
What's the benefit of `lodash` over native `.trim()`?

Reply:
Be consistent in the usage of these auxiliary functions? I've seen we use to use lodash even when it's supported in a native way.

For instance using `noop()` instead of `() => {}`.
Anyway, it's a minor suggestion. :-)

Reply:
I think using lodash's `trim()` would be an overkill here (and in most other places, probably) - the fewer dependencies the better, IMHO. I guess that in some specific cases, where the performance is crucial we should be looking for alternatives if they do the job better, but this is not one of those cases.

Reply:
I agree with @WunderBart and I left it as-is for now. 

IMHO, native solutions are usually easier to read for most developers.